storage-api/pom.xml

```diff
@@ -118,6 +118,7 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
       <version>${guava.version}</version>
+      <scope>test</scope>
```

Comment:
This was removed here without the reason of removal. I hope this recovers the dependency without breaking CIs.
- https://github.com/apache/hive/pull/1784/files#diff-e5e45b8481f3d87f4fa561d45b22fa20bb5084e5240d825717e6bde511b733baL122

If some CI fails, it means the regression already spread to the other modules.